reviewer_config.prototxt

```diff
@@ -0,0 +1,93 @@
+display_name: "StartupOS"
+repo {
+  id: "startup-os"
+  url: "https://github.com/google/startup-os"
+}
+repo {
+  id: "hasadna"
+  url: "https://github.com/hasadna/hasadna"
+}
+project {
```

Comment:
I'd remove all projects because it relates to StartupOS. Instead of it add:
// TODO: Add hasadna's projects

Reply:
Good idea.